conda_smithy/linter/hints.py

```diff
@@ -232,3 +232,29 @@ def hint_pip_no_build_backend(host_or_build_section, package_name, hints):
                 "If your recipe has built with only `pip` in the `host` section in the past, you likely should "
                 "add `setuptools` to the `host` section of your recipe."
             )
+
+
+def hint_noarch_python_use_python_min(
+    host_reqs, run_reqs, test_reqs, outputs_section, noarch_value, hints
+):
+    if noarch_value == "python" and not outputs_section:
```

Comment:
We don't hint multi output recipes? 🤔

Reply:
Not currently no. IDK why.

Reply:
@isuruf added similar code in https://github.com/conda-forge/conda-smithy/commit/b7514aa03125d7dcc13de6da1206577c34ac69bf

There's no PR attached to that blame so I'm not sure where it came from or whether there was some discussion. We should probably make it compatible with multi output recipes but I won't block if you consider it's out of scope.

Reply:
I think it is out of scope for this PR given that it is already in the code base.